wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -851,24 +860,14 @@ public void onCreate() {
 
         headerChainFile = new File(getDir("blockstore", Context.MODE_PRIVATE), Constants.Files.HEADERS_FILENAME);
 
-        try {
-            bootStrapStream = getAssets().open(Constants.Files.MNLIST_BOOTSTRAP_FILENAME);
-            SimplifiedMasternodeListManager.setBootStrapStream(bootStrapStream, SimplifiedMasternodeListManager.QUORUM_ROTATION_FORMAT_VERSION);
-        } catch (IOException x) {
-            log.info("cannot load the boot strap stream.  " + x.getMessage());
-        }
+        mnlistinfoBootStrapStream = loadStream(Constants.Files.MNLIST_BOOTSTRAP_FILENAME);
+        qrinfoBootStrapStream = loadStream(Constants.Files.QRINFO_BOOTSTRAP_FILENAME);
```

Comment:
There are effectively two masternode/quorum synchronization systems running in parallel, so we need two bootstraps, one for each.